### ISSUE TITLE
Dependency 'libssl1.0-dev' does not work on Ubuntu 16

### ISSUE THREAD

**Issue by Luk3M** [r888888888/danbooru#3481]
Trying to run `INSTALL.debian` on Ubuntu 16 does not work because it can't find `libssl1.0-dev`.

Changing it to `libssl-dev` on both `INSTALL.debian` and `install.sh` makes it work again.

It looks like there is a condition that is testing for which package to use, but it does not seems like is working properly for Ubuntu 16.

**Comment by BrokenEagle** [comment]
Really?  I'm running Ubuntu-16.04.2-Server-AMD64, and I don't recall any issues with installing that particular library, although that was several months ago.

Also, I just checked ``apt list --installed`` and saw that the following had been installed:
```
libssl-dev/xenial-updates,xenial-security,now 1.0.2g-1ubuntu4.10 amd64 [installed]
```

I also checked [Line 36](https://github.com/r888888888/danbooru/blob/0bb2dae5f6c067f88f5ffe4ff7976708bbb9caa6/INSTALL.debian#L36) of the install file and saw that it was setting ``LIBSSL_DEV_PKG`` to "libssl1.0-dev".
